control_node:
  ros__parameters:
    cmd_vel_topic: "cmd_vel"
    encoder_topic: "encoder_data"
    odom_topic: "odom"
    wheel_separation: 0.788
    wheel_radius: 0.11736
    left_wheel_names: ["front_left_wheel_joint", "back_left_wheel_joint"]
    right_wheel_names: ["front_right_wheel_joint", "back_right_wheel_joint"]